config/examples/Sovol/SV-01/BigTreeTech SKR Mini E3 3.0/Fixed Probe/Configuration.h

```diff
@@ -0,0 +1,3106 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (c) 2021 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ * Based on Sprinter and grbl.
+ * Copyright (c) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * 
+ */
+#pragma once
+
```

Comment:
Did you start with a `bugfix-2.0.x` config by chance? You're missing these lines as well:

```cpp
#error "Don't build with import-2.0.x configurations!"
#error "Use the 'bugfix...' or 'release...' configurations matching your Marlin version."
```

Reply:
Yes I started with bugfix-2.0.x

Lines added

Reply:
Files replaced with import-2.0.x configs